### PR TITLE
Updating Dev

### PR DESCRIPTION
**Pull request by RolivhuwaMuzila**
Merging main into Dev to update Dev

### CODE REVIEW

**Comment by asandan8106** [comment]
Nice.

**Comment by Xolanisyav04** [comment]
great work!! I like the styling and also great functionality.

**Comment by Pali-Pali** [comment]
I like your chatbot design especially the change of modes (dark and light mode).

File: Chatbot.js [hunk 1]
```diff
@@ -16,9 +16,22 @@ const portfolioInfo = {
         "Login Authentication",
         "Movie Recommend"
```
Comment (atlehang66) [inline Chatbot.js]: Remove this line " movie recommend"

File: Chatbot.js [hunk 2]
```diff
@@ -16,9 +16,22 @@ const portfolioInfo = {
         "Login Authentication",
         "Movie Recommend"
     ],
-    skills: ["HTML", "CSS", "JavaScript", "React", "Node.js", "Python"],
```
Comment (atlehang66) [inline Chatbot.js]: Great implementation of chat bot logic. I approve. keep it up!!!!

File: Chatbot.js (hunk 1 above)
Comment (RolivhuwaMuzila) [reply Chatbot.js]: I have removed the "movie recommend" already, Thank you .

**Pull request merged**